### ISSUE TITLE
\core_privacy\local\metadata\null_provider is not appropriate provider

### ISSUE THREAD

**Issue by mudrd8mz** [center-for-learning-management/moodle-mod_msteams#11]
The plugin shares user's preferred language with an external system (it passes it as a `local` argument to the `meetingapplink`. As such, it should indicate that as per https://docs.moodle.org/dev/Privacy_API#Indicating_that_you_export_data_to_an_external_location so that admins can incorporate that information into their site policies.

**Comment by mudrd8mz** [comment]
BTW, I don't know details of the remote system implementation. Just note that `$USER->lang` hold the Moodle's internal language pack code. You may want to use the standard language code, e.g. `get_string('iso6391', 'localconfig')` or get_string('iso6392', 'localconfig')`

**Comment by rschrenk** [comment]
@mudrd8mz thanks for that hint. I took the null provider as it is also used in the plugin atto_teamsmeeting that was the original plugin. I informed the developers of atto_teamsmeeting about that issue too (https://github.com/enovation/moodle-atto_teamsmeeting/issues/28).

The implementation to use $USER->lang is also from them, so I think this is ok.

The updated privacy provider is implemented in 6034b00cdb0b2071ba07b19d48771d6c56c4952a and will be published on moodle.org with the next plugin release.
